src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/SubscriptionController.cs

```diff
@@ -0,0 +1,306 @@
+
+                if (!ssn.Equals(eventsSubscription.AlternativeSubjectFilter))
+                {
+                    return false;
+                }
+            }
+            else if (!string.IsNullOrEmpty(eventsSubscription.AlternativeSubjectFilter) && !eventsSubscription.AlternativeSubjectFilter.Equals(eventsSubscription.Consumer))
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        private async Task<string> GetPartyFromAlternativeSubject(string alternativeSubject)
+        {
+            int partyId = 0;
+
+            if (alternativeSubject.StartsWith(OrganizationPrefix))
+            {
+                string orgNo = alternativeSubject.Replace(OrganizationPrefix, string.Empty);
+                partyId = await _registerService.PartyLookup(orgNo, null);
+            }
+            else if (alternativeSubject.StartsWith(PersonPrefix))
+            {
+                string persnoNo = alternativeSubject.Replace(PersonPrefix, string.Empty);
+                partyId = await _registerService.PartyLookup(null, persnoNo);
+            }
+
+            if (partyId != 0)
+            {
+                return "/party/" + partyId;
+            }
+
+            return null;
+        }
+
+        private void EnrichConsumer(EventsSubscription eventsSubscription)
+        {
+            if (string.IsNullOrEmpty(eventsSubscription.Consumer))
+            {
+                string org = HttpContext.User.GetOrg();
+                if (!string.IsNullOrEmpty(org))
+                {
+                    eventsSubscription.Consumer = "/org/" + org;
+                    return;
+                }
+
+                int? userId = HttpContext.User.GetUserIdAsInt();
+                if (userId.HasValue)
+                {
+                    eventsSubscription.Consumer = "/user/" + userId.Value;
+                    return;
+                }
+
+                string organization = HttpContext.User.GetOrgNumber();
+                if (!string.IsNullOrEmpty(organization))
+                {
+                    eventsSubscription.Consumer = "/organization/" + organization;
+                    return;
+                }
+            }
+        }
+
+        private void SetCreatedBy(EventsSubscription eventsSubscription)
+        {
+            string org = HttpContext.User.GetOrg();
+            if (!string.IsNullOrEmpty(org))
+            {
+                eventsSubscription.CreatedBy = "/org/" + org;
+                return;
+            }
+
+            int? userId = HttpContext.User.GetUserIdAsInt();
+            if (userId.HasValue)
+            {
+                eventsSubscription.CreatedBy = "/user/" + userId.Value;
+                return;
+            }
+
+            string organization = HttpContext.User.GetOrgNumber();
+            if (!string.IsNullOrEmpty(organization))
+            {
+                eventsSubscription.CreatedBy = "/organization/" + organization;
```

Comment:
I don't quite remember what we concluded with. But I  this we should double check using organization with a `z` in this string.. We have concluded with british spelling so it should be organisation with an  s for everything that is exposed to the end user.. this will be a part of the response object for whoever tries to retrieve an event so should proabably be spelled with an s.

Reply:
I'd like to see that the OrganizationPrefix  prop is used here rather than writing it out as well

Reply:
Updated. Need to verify if we do this in app. The example in docs uses z